tests/tests_app/utilities/test_commands.py

```diff
@@ -97,7 +97,8 @@ def test_validate_client_command():
     with pytest.raises(Exception, match="annotate your method"):
         _validate_client_command(ClientCommand(run_failure_1))
 
-    with pytest.raises(Exception, match="lightning_app/utilities/commands/base.py"):
+    starts = "lightning_app".replace(".", "/")
```

Comment:
same question here

Reply:
It's probably best to grep lightning_app over the whole folder to see if there are any leftovers?

Reply:
This is also because of the adjustments in CI which has been there since beginning
and the search and changes will be carried in #16554